packages/clerk-js/src/ui/elements/InformationBox.tsx

```diff
@@ -0,0 +1,26 @@
+import type { LocalizationKey } from '../customizables';
+import { Flex, Icon, Text } from '../customizables';
+import { InformationCircle } from '../icons';
+
+type InformationBoxProps = {
+  message: LocalizationKey | string;
+};
+
+export function InformationBox(props: InformationBoxProps) {
```

Comment:
This should ideally have/accept descriptor keys in order to generate the appropriate classes.

Reply:
Let's do it once requested :)